stixcore/util/logging.py

```diff
@@ -20,10 +20,14 @@
 
 """
 import logging
+import warnings
 
 STX_LOGGER_FORMAT = '%(asctime)s %(levelname)s %(name)s %(lineno)s: %(message)s'
 STX_LOGGER_DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
+warnings.filterwarnings('ignore', module='astropy.io.fits.card')
+warnings.filterwarnings('ignore', module='astropy.utils.metadata')
+
```

Comment:
I'm not sure this is the best place for this as this will turn off in all modules prob only should do in the pipeline?

Reply:
moved